<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>scWizardWgt</class>
 <widget class="QWidget" name="scWizardWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SC Wizard</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="ScAllTypesWidget" name="allTypes"/>
     <widget class="ScTokenWgt" name="tokenProperties"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScAllTypesWidget</class>
   <extends>QWidget</extends>
   <header>scalltypeswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScTokenWgt</class>
   <extends>QWidget</extends>
   <header>sctokenwgt.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
